cupy_setup_build.py

```diff
@@ -947,14 +947,14 @@ def cythonize(extensions, arg_options):
     compile_time_env['CUPY_CUFFT_STATIC'] = False
     compile_time_env['cython_version'] = str(cython_version)
```

Comment:
Also rename to `cupy_cython_version` or `CUPY_CYTHON_VERSION`?

Reply:
Sounds good!